.github/workflows/cdktf-provider-docs-rollout.yml

```diff
@@ -0,0 +1,61 @@
+name: CDKTF Provider Docs Rollout
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: 0 0 * * 1
+
+jobs:
+  cdktfDocs:
+    strategy:
+      fail-fast: false
+      matrix:
+        provider:
+          # To onboard new providers, add a new entry to this matrix and make sure the team-tf-cdk github user has write permissions to the repository.
+          - repo: hashicorp/terraform-provider-archive
+            fqn: "hashicorp/archive"
+          - repo: hashicorp/terraform-provider-cloudinit
+            fqn: "hashicorp/cloudinit"
+          - repo: hashicorp/terraform-provider-dns
+            fqn: "hashicorp/dns"
+          - repo: hashicorp/terraform-provider-external
+            fqn: "hashicorp/external"
+          - repo: hashicorp/terraform-provider-http
+            fqn: "hashicorp/http"
+          - repo: hashicorp/terraform-provider-local
+            fqn: "hashicorp/local"
+          - repo: hashicorp/terraform-provider-null
+            fqn: "hashicorp/null"
+          - repo: hashicorp/terraform-provider-random
+            fqn: "hashicorp/random"
+          - repo: hashicorp/terraform-provider-time
+            fqn: "hashicorp/time"
+          - repo: hashicorp/terraform-provider-tls
+            fqn: "hashicorp/tls"
+
+    uses: hashicorp/terraform-cdk/.github/workflows/registry-docs-pr-based.yml@1b1be8ee471bb2845356e064e9f10b484f636cd9
```

Comment:
(optional) Do we need to pin if they point to the same repo? I could see future me trying to wonder why changes in the other workflow are not reflected 😅

Reply:
No idea, I can try the local syntax and see if it works for security. But probably should